src/components/Button/Button.js

```diff
@@ -142,7 +144,11 @@ class Button extends Component {
     }
 
     handleRef = (ref) => {
-        this.componentRef = ref;
+        const { componentRef } = this.props;
+        this.buttonRef = ref;
```

Comment:
`this.componentRef` wasn't use, so I took the opportunity to rename it to `buttonRef` to avoid any confusion